saleor/graphql/payment/mutations/transaction/transaction_event_report.py

```diff
@@ -81,7 +82,7 @@ class Arguments:
             description="Current status of the event to report.",
         )
         amount = PositiveDecimal(
-            description="The amount of the event to report.", required=True
+            description="The amount of the event to report.", required=False
```

Comment:
I'm wondering if the information about when `amount` is required should be included in description. WDYT?

Reply:
Good point!

Reply:
I updated it